src/helpers/client.js

```diff
@@ -0,0 +1,18 @@
+import { config } from 'config'
```

Comment:
This will fail because there is no installed `config` package. It's a local file.
```suggestion
import { config } from '../config'
```